src/App.js

```diff
@@ -1,24 +1,15 @@
-import logo from './logo.svg';
 import './App.css';
```

Comment:
Can we create an App folder inside components and move app.js, app.css into that folder?

Reply:
Sure!